### PR TITLE
primeira tradução 

### PR DESCRIPTION
**Pull request by raphaelpe**
traduzindo a primeira linha

### CODE REVIEW

**Comment by nrotstan** [comment]
Thank you for working on this, @raphaelpe!

**Comment by nrotstan** [comment]
This is awesome, @raphaelpe! Is it finished and ready to be merged into the project, or are you still working on it?

**Comment by raphaelpe** [comment]
Hello Nrotstan, Good evening, I made the translation of every file more appeared in red lines and others in green, what would it be? Does some kind of review need to be done? If you need to do I do, did you do any revision? I await your response! I would like you to send me an e-mail from your staff to send you some requests when it comes to the maproulette project?

**Comment by raphaelpe** [comment]
Good Night nrotstan, I wonder if a revision of the translation I did was made?

File: src/lang/pt_BR.json [hunk 1]
```diff
@@ -0,0 +1,751 @@
...
+  "Challenge.sort.created": "Newest",
+  "Challenge.sort.popularity": "Popular",
+  "Challenge.sort.default": "Padrão",
+  "Task.loadByMethod.random": "Aleatória",
+  "Task.loadByMethod.proximity": "Proximidade",
+  "Task.priority.high": "Alto",
+  "Task.priority.medium": "Médio",
+  "Task.priority.low": "Baixo",
+  "Task.status.created": "Criado",
+  "Task.status.fixed": "Fixo",
```
Comment (willemarcel) [inline src/lang/pt_BR.json]: Fixed aqui significa Corrigido

File: src/lang/pt_BR.json [hunk 2]
```diff
@@ -0,0 +1,751 @@
...
+  "Task.status.falsePositive": "Não é um problema",
+  "Task.status.skipped": "Pulou",
+  "Task.status.deleted": "Excluído",
+  "Task.status.alreadyFixed": "Já fixa",
```
Comment (willemarcel) [inline src/lang/pt_BR.json]: Já corrigido

File: src/lang/pt_BR.json [hunk 3]
```diff
@@ -0,0 +1,751 @@
...
+  "Locale.es.label": "es (Español)",
+  "Locale.de.label": "de (Deutsch)",
+  "Locale.fr.label": "fr (Français)",
+  "Locale.af.label": "af (Afrikaans)",
+  "Locale.ja.label": "ja (日本語)",
+  "Locale.ko.label": "ko (한국어)",
+  "Dashboard.ChallengeFilter.visible.label": "Visível",
+  "Dashboard.ChallengeFilter.pinned.label": "Preso",
+  "Dashboard.ProjectFilter.visible.label": "Visível",
+  "Dashboard.ProjectFilter.owner.label": "Possuído",
```
Comment (willemarcel) [inline src/lang/pt_BR.json]: owner = Dono ou Proprietário

File: src/lang/pt_BR.json [hunk 4]
```diff
@@ -0,0 +1,751 @@
...
+  "KeyMapping.layers.layerTaskFeatures": "Alternar Camada de Recursos",
+  "KeyMapping.layers.layerMapillary": "Alternar Camada Mapilar",
+  "KeyMapping.taskEditing.cancel": "Cancelar edição",
+  "KeyMapping.taskEditing.fitBounds": "Ajustar Mapa aos Recursos da Tarefa",
+  "KeyMapping.taskEditing.escapeLabel": "ESC",
+  "KeyMapping.taskCompletion.skip": "Pular",
+  "KeyMapping.taskCompletion.falsePositive": "Não é um problema",
+  "KeyMapping.taskCompletion.fixed": "Eu consertei isso!",
+  "KeyMapping.taskCompletion.tooHard": "Muito difícil / não consegui ver",
+  "KeyMapping.taskCompletion.alreadyFixed": "Já fixa",
```
Comment (willemarcel) [inline src/lang/pt_BR.json]: Já corrigido

File: src/lang/pt_BR.json [hunk 5]
```diff
@@ -0,0 +1,751 @@
...
+  "Challenge.status.failed": "Falhou",
+  "Challenge.status.ready": "Pronto",
+  "Challenge.status.partiallyLoaded": "Parcialmente carregado",
+  "Challenge.status.finished": "Terminado",
+  "Challenge.type.challenge": "Desafio",
+  "Challenge.type.survey": "Inquérito",
+  "Editor.none.label": "Nenhum",
+  "Editor.id.label": "Editar no iD (editor da web)",
+  "Editor.josm.label": "Editar no JOSM",
+  "Editor.josmLayer.label": "Editar na nova camada do JOSM",
```
Comment (willemarcel) [inline src/lang/pt_BR.json]: trocar "na" por "em uma"

File: src/lang/pt_BR.json [hunk 6]
```diff
@@ -0,0 +1,751 @@
...
+  "Challenge.location.intersectingMapBounds": "Interseção de limites do mapa",
+  "Challenge.location.any": "Em qualquer lugar",
+  "Challenge.status.none": "Não aplicável",
+  "Challenge.status.building": "Edifício",
+  "Challenge.status.failed": "Falhou",
+  "Challenge.status.ready": "Pronto",
+  "Challenge.status.partiallyLoaded": "Parcialmente carregado",
+  "Challenge.status.finished": "Terminado",
+  "Challenge.type.challenge": "Desafio",
+  "Challenge.type.survey": "Inquérito",
```
Comment (willemarcel) [inline src/lang/pt_BR.json]: Pesquisa

File: src/lang/pt_BR.json [hunk 7]
```diff
@@ -0,0 +1,751 @@
...
+  "Activity.action.taskViewed": "Visto",
+  "Activity.action.taskStatusSet": "Definir status no",
+  "Activity.action.tagAdded": "Tag adicionado para",
+  "Activity.action.tagRemoved": "Tag removido de",
+  "Activity.action.questionAnswered": "Pergunta respondida em",
+  "Activity.item.project": "Projeto",
+  "Activity.item.challenge": "Desafio",
+  "Activity.item.task": "Tarefa",
+  "Activity.item.tag": "Tag",
+  "Activity.item.survey": "Inquérito",
```
Comment (willemarcel) [inline src/lang/pt_BR.json]: Pesquisa

**Comment by nrotstan** [comment]
Hi @raphaelpe, I have looked at it briefly, but have not yet tested it. I will do that soon. :-) The current plan is to include it in the next release (v3.2.4)

File: src/lang/pt_BR.json (hunk 1 above)
Comment (raphaelpe) [reply src/lang/pt_BR.json]: Corrigi conforme sugestão! Obrigado! / Correct as per suggestion! Thank you!

File: src/lang/pt_BR.json (hunk 3 above)
Comment (raphaelpe) [reply src/lang/pt_BR.json]: Corrigi conforme sugestão! Obrigado! / Correct as per suggestion! Thank you!

File: src/lang/pt_BR.json (hunk 5 above)
Comment (raphaelpe) [reply src/lang/pt_BR.json]: Corrigi conforme sugestão! Obrigado! / Correct as per suggestion! Thank you!

File: src/lang/pt_BR.json (hunk 6 above)
Comment (raphaelpe) [reply src/lang/pt_BR.json]: Corrigi conforme sugestão! Obrigado! / Correct as per suggestion! Thank you!

File: src/lang/pt_BR.json (hunk 7 above)
Comment (raphaelpe) [reply src/lang/pt_BR.json]: Corrigi conforme sugestão! Obrigado! / Correct as per suggestion! Thank you!

**Comment by raphaelpe** [comment]
nrotstan, a comunidade brasileira agradece! /  nrotstan, the Brazilian community thanks!

**Pull request closed without merging**